File: Javascript/BubbleSort.js
function bubbleSort (array) {
    let sorted = false
    while (sorted !== true) {
        let swaps = 0
        for (let counter = 0; counter < array.length - 1; counter += 1) {
            if (array[counter] > array[counter + 1]) {
                let lastItem = array[counter]
                array[counter] = array[counter + 1]
                array[counter + 1] = lastItem
                swaps += 1
            }
        }
        if (swaps === 0) sorted = true
    }
    return array
}

File: Python/BubbleSort.py
def bubbleSort(arr): 
    #Loops through all elements
    for i in range(len(arr)):
        #Last elements are already placed 
        for j in range(0, len(arr)-i-1):
            if arr[j] > arr[j+1] : 
                arr[j], arr[j+1] = arr[j+1], arr[j] 
  
arr = [86,59,62,72,48,30,45] 
  
bubbleSort(arr) 
  
print ("Sorted array is:",end = " ")
print(arr)